### PR TITLE
SG-34074 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the ShotGrid Pipeline Toolkit.
+This repository is a part of the Flow Production Tracking Toolkit Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
 - For general information and documentation, click here: https://help.autodesk.com/view/SGDEV/ENU/?contextId=SA_INTEGRATIONS_USER_GUIDE
-- For information about ShotGrid in general, click here: https://www.autodesk.com/products/shotgrid/overview
+- For information about Flow Production Tracking Toolkit in general, click here: https://www.autodesk.com/products/shotgrid/overview
```
Comment (julien-lang) [inline README.md:12]: I don't think this refers to Toolkit but to SG/Flow. So I think this (and similar ones below) should be:

```patch
- ShotGrid
+ Flow Production Tracking
```

File: README.md [hunk 2]
```diff
@@ -5,11 +5,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the ShotGrid Pipeline Toolkit.
+This repository is a part of the Flow Production Tracking Toolkit Pipeline Toolkit.
```
Comment (julien-lang) [inline README.md:8]: Well, we are not going to repeat the toolkit twice: `Flow Production Tracking Toolkit Pipeline Toolkit`

But the question is what do we do with "Pipeline Toolkit"? I see two ways:

- Forget the word pipeline: Flow Production Tracking Toolkit
- Append the work pipeline: Flow Production Tracking Toolkit Pipeline

I'd prefer option 1. But we should ask around.

**Review by julien-lang — COMMENTED** [review]
The Maya icon would have to be updated too. But we need to wait for the asset.

File: README.md (hunk 2 above)
Comment (julien-lang) [reply README.md:8]: @juanburgosautoglb I got confirmation from Eric: option 1. We remove the pipeline word.

**Comment by julien-lang** [comment]
> The Maya icon would have to be updated too. But we need to wait for the asset.

No, no need for that.

File: SECURITY.md [hunk 3]
```diff
@@ -4,18 +4,18 @@
 
 At Autodesk, we know that the security of your data is critical to your studio’s
 operation.
-As the industry shifts to the cloud, ShotGrid knows that security and service
+As the industry shifts to the cloud, Flow Production Tracking Toolkit knows that security and service
```
Comment (julien-lang) [inline SECURITY.md:7]: remove toolkit

File: SECURITY.md [hunk 4]
```diff
@@ -4,18 +4,18 @@
...
 models are more important than ever.
 
 The confidentiality, integrity, and availability of your content is at the top
 of our priority list.
-Not only do we have a team of ShotGrid engineers dedicated to platform security
+Not only do we have a team of Flow Production Tracking Toolkit engineers dedicated to platform security
```
Comment (julien-lang) [inline SECURITY.md:12]: remove toolkit

File: SECURITY.md [hunk 5]
```diff
@@ -4,18 +4,18 @@
...
 and performance, we are also backed by Autodesk’s security team, also invests
 heavily in the security for broad range of industries and customers.
 We constantly reassess, develop, and improve our risk management program because
 we know that the landscape of security is ever-changing.
 
-If you believe you have found a security vulnerability in any ShotGrid-owned
+If you believe you have found a security vulnerability in any Flow Production Tracking Toolkit-owned
```
Comment (julien-lang) [inline SECURITY.md:18]: remove toolkit

File: SECURITY.md [hunk 6]
```diff
@@ -32,4 +32,4 @@ configurations, reproduction steps, exploit code, impact, etc.
 
 ## Additional Information
 
-Please check out the [ShotGrid Security White Paper](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_general_security_ar_security_white_paper_html).
+Please check out the [Flow Production Tracking Toolkit Security White Paper](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_general_security_ar_security_white_paper_html).
```
Comment (julien-lang) [inline SECURITY.md:35]: remove toolkit

File: engine.py [hunk 7]
```diff
@@ -469,7 +471,7 @@ def init_engine(self):
 
         # default menu name is Shotgun but this can be overriden
         # in the configuration to be Sgtk in case of conflicts
-        self._menu_name = "ShotGrid"
+        self._menu_name = "Flow Production Tracking Toolkit"
```
Comment (julien-lang) [inline engine.py:474]: remove toolkit

File: engine.py [hunk 8]
```diff
@@ -568,7 +569,7 @@ def _run_app_instance_commands(self):
 
         # Build a dictionary mapping app instance names to dictionaries of commands they registered with the engine.
         app_instance_commands = {}
-        for (command_name, value) in self.commands.items():
+        for command_name, value in self.commands.items():
```
Comment (julien-lang) [inline engine.py:572]: useless change

File: engine.py [hunk 9]
```diff
@@ -579,7 +580,6 @@ def _run_app_instance_commands(self):
 
         # Run the series of app instance commands listed in the 'run_at_startup' setting.
         for app_setting_dict in self.get_setting("run_at_startup", []):
-
```
Comment (julien-lang) [inline engine.py:582]: useless change

File: engine.py (hunk 8 above)
Comment (julien-lang) [reply engine.py:572]: multiple iterations in this PR. please clean them all

File: engine.py (hunk 9 above)
Comment (julien-lang) [reply engine.py:582]: multiple iterations in this PR. please clean them all

File: plugins/basic/README.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Maya Basic Toolkit workflow plugin
 
-This is a ShotGrid Pipeline Toolkit plugin,
-embedding the ShotGrid Pipeline Toolkit and allowing
+This is a Flow Production Tracking Toolkit Pipeline Toolkit plugin,
+embedding the Flow Production Tracking Toolkit Pipeline Toolkit and allowing
```
Comment (julien-lang) [inline plugins/basic/README.md:4]: Remove pipeline toolkit. Multiple iterations. Please clean them all

File: info.yml [hunk 11]
```diff
@@ -88,8 +88,8 @@ configuration:
 requires_shotgun_fields:
 
 # More verbose description of this item
-display_name: "ShotGrid Engine for Maya"
-description: "ShotGrid Integration in Maya"
+display_name: "Flow Production Tracking Toolkit Engine for Maya"
```
Comment (julien-lang) [inline info.yml:91]: Remove Toolkit. Multiple iterations. Please clean them all

**Review by julien-lang — COMMENTED** [review]
Looks good. Please fix CI tests

**Review by NorberMV — APPROVED** [review]
LGTM!

**Pull request merged**